testnets/cosmwasmtestnet/assetlist.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "../../assetlist.schema.json",
+  "chain_name": "malaga-420",
+  "assets": [
+    {
+      "description": "Fee Token",
+      "denom_units": [
+        {
+          "denom": "umlg",
+          "exponent": 0
+        },
+        {
+          "denom": "MLG",
+          "exponent": 6
+        }
+      ],
+      "base": "umlg",
+      "name": "Malaga Testnet",
```

Comment:
These name are meant to describe the token, rather than the chain... both tokens are to be called "Malaga Testnet"?

Reply:
likely Malaga — I'll pass these updates on to Milan